packages/nimble-components/src/label-provider/base/label-provider-stories-utils.ts

```diff
@@ -0,0 +1,95 @@
+import { ViewTemplate, html, repeat } from '@microsoft/fast-element';
+import type { Meta } from '@storybook/html';
+import type { DesignToken } from '@microsoft/fast-foundation';
+import { createUserSelectedThemeStory } from '../../utilities/tests/storybook';
+import {
+    bodyFont,
+    groupHeaderFont,
+    groupHeaderFontColor,
+    groupHeaderTextTransform
+} from '../../theme-provider/design-tokens';
+import { getAttributeName, getPropertyName } from './label-name-utils';
+
+export interface LabelProviderArgs {
+    labelProviderTag: string;
+    labelTokens: [string, DesignToken<string>][];
+    removeNamePrefix(tokenName: string): string;
+}
+
+const createTemplate = (
+    labelProviderTag: string
+): ViewTemplate<LabelProviderArgs> => html<LabelProviderArgs>`
+<${labelProviderTag}></${labelProviderTag}>
+<p>Element name: <code>${x => x.labelProviderTag}</code></p>
+<table>
```

Comment:
Can you comment why you ended up using a native table and adding nimble-table styling to it? It looks a bit janky to me. My main complaints are the lack of row delineation and the inconsistent text alignment between headers and cells. Those would be easy enough to fix, but could we instead consider just using the nimble-table like we did for the component status story?

Edit: I see the theme aware token story is also using native tables. I think that's needed because of the varied contents of the token value column, but we don't have the same requirement for these tables.

Reply:
Yep I started by adapting the theme-aware token story. I can try out using Nimble table instead.

Reply:
Switched to nimble-table